lib/logstash/outputs/influxdb.rb

```diff
@@ -29,18 +32,18 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   config :port, :validate => :number, :default => 8086
 
   # The user who has access to the named database
-  config :user, :validate => :string, :default => nil, :required => true
+  config :user, :validate => :string, :default => nil
 
   # The password for the user who access to the named database
-  config :password, :validate => :password, :default => nil, :required => true
+  config :password, :validate => :password, :default => nil
 
-  # Series name - supports sprintf formatting
-  config :series, :validate => :string, :default => "logstash"
+  # Measurement name - supports sprintf formatting
+  config :measurement, :validate => :string, :default => "logstash"
```

Comment:
breaking change, renaming 'series' to 'measurement'

We'll want to bump the major version of this plugin as a result.